### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by vrmiguel**
This PR focuses on improving the legibility of the README, alongside minor typo and grammar fixes.

Notes: 
* I assumed that Woori refers to a Korean word. If I'm wrong, please let me know.
* I didn't yet go over the several SELECT request descriptions but probably will later on.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,26 +1,29 @@
...
-  - [ ] Non sensible configurations with `Config.toml`.
+  - [ ] Non sensitive configurations are done with `Config.toml`.
   - [ ] CORS
 - Authentication and Authorization via session token
   - [ ] Creating and removing ADMINs/new users.
 - [Conditional Update](https://github.com/naomijub/wooridb#match-update-entity)
 - [ ] Possible Relation Algebra
 
-`Woori` means `our` and although I developed this DB initially alone, it is in my culture to call everything that is done for our community and by our community **ours**.
+`Woori` (우리) means `our` and although I developed this DB initially alone, it is in my culture to call everything that is done for our community and by our community **ours**.
```
Comment (naomijub) [inline README.md:26]: I just don't want it to have the hangul version. This is a personal reason, as well as it has a similar sounding to altaic tatar language that has sounds that variates from Biz to Bwoozi (also personal, but way less known).

**Review by naomijub — CHANGES_REQUESTED** [review]

**Comment by naomijub** [comment]
> I didn't yet go over the several SELECT request descriptions but probably will later on.
It is just one `### Select`, I should have done the examples with `####`. Will fix this soon, thanks.

**Comment by naomijub** [comment]
Also, thanks on the congrats and the reviews.

**Comment by vrmiguel** [comment]
I removed the hangul and added some other minor stuff

File: README.md [hunk 2]
```diff
@@ -216,7 +216,7 @@ To better understand the next sub-items, lets say the entity `my_entity_name` ha
 
 
 ### SELECTS all keys FROM ENTITY:
-Select All entities from `my_entity` with all keys for each entity. 
+Select sll entities from `my_entity` with all the keys from each entity.
```
Comment (naomijub) [inline README.md:219]: `sll`

Comment (naomijub) [reply README.md:219]: @vrmiguel one more thing

**Comment by vrmiguel** [comment]
Fixed the 'sll' typo

**Review by naomijub — APPROVED** [review]

**Pull request merged**